documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+
+In either case, it is still courteous to check with the individuals and make it
+clear that they can push the discussion to someone else (provided they're not a
+necessary subject-matter expert).
+
+#### Project level process discussions
+
+Because project level process discussions usually effect all maintainers and
+sometimes other contributors, they naturally garner the expectation of
+involvement from broader sets of people than technical discussions. In addition
+to the guidelines above for general discussions, we need additional special
+guidelines to ensure we do not become over encumbered with process discussions
+specifically. In particular, retrospectives may give rise to several discussions
+that need to happen at once. In these cases, and others where process
+discussions begin to dominate the discussion backlog, we'll need to triage
+discussions. If discussions truly need the involvement of the entire team, we
+can follow two processes:
+
+1. Put (and prioritise) the discussions into the "proposal requested" column of
+   the discussion dashboard and only allow one team-wide discussion to occur at
+   a time. Limiting this to 1 allows other non-team-wide discussions to also
+   happen at the same time without flooding the discussion board with team-wide
+   discussions and halting other, smaller, often technical discussions.
+2. Split the group into groups of three or four people who can participate in
+   different parts of the discussion. This is useful if there are multiple
+   time-sensitive discussions and could lead to synchronous "lightening process"
+   meetings to make faster decisions and reduce the ongoing discussion burden on
+   team members. Even in this case, limit to 1 ongoing team-wide process
+   discussion per group of contributors. This is less useful if everyone on the
+   team wishes to give input in the discussion, although people are free to
+   asynchronously give input before the fact.
+
+For both, we will aim to limit the process discussions to one discussion at a
+time for any given team member. This is recommended to avoid discussion burn out
+as I think I’ve observed process discussions feeling much heavier weight and
+burdensome than technical discussions. It also makes sure there is still room
+for team members to be involved in one other separate technical discussion and
+feel like they can still focus on their other work duties.
+
+### Speeding up decision-making
+
+The base process should help the Openverse project make decisions more quickly
+and effectively. However, the following additional practices are available to
+help move things along at an even faster clip, if so desired.
+
+#### "Shortcutting" a round
+
+At any point during a round, any participant may say something along the lines
+of "no (further) questions", "no reaction", or "no objection" in order to
+short-cut their participation in the round. If all participants are finished
```

Comment:
Here, too, I think it would be useful to "Approve the PR" after leaving a note like "no objection" to make the merging later easier.